tests/functional/workloads/ocp/monitoring/test_monitoring_on_negative_scenarios.py

```diff
@@ -465,8 +465,18 @@ def test_monitoring_after_rebooting_node_where_mgr_is_running(
         # Get the mgr pod obj
         mgr_pod_obj = pod.get_mgr_pods()
 
+        # Get active mgr pod
+        toolbox = pod.get_ceph_tools_pod()
+        active_mgr_pod_output = toolbox.exec_cmd_on_pod("ceph mgr stat")
+        active_mgr_pod_suffix = active_mgr_pod_output.get("active_name")
+        log.info(f"The active MGR pod is {active_mgr_pod_suffix}")
+        for obj in mgr_pod_obj:
+            if active_mgr_pod_suffix in obj.name:
+                active_mgr_pod_obj = obj
+        log.info(f"The active MGR pod name is  {active_mgr_pod_obj.name}")
+
         # Get the node where the mgr pod is hosted
-        mgr_node_obj = pod.get_pod_node(mgr_pod_obj[0])
+        mgr_node_obj = pod.get_pod_node(active_mgr_pod_obj)
```

Comment:
we can also use label `mgr_role=active` instead of running ceph cmd

Reply:
Okay. Can I change it in next PR as it is not affecting tests functionality?

Reply:
Okay. Can I change it in next PR as it is not affecting tests functionality?

Reply:
please create a issue to track this